src/Congo.WebApi/Data/ProductAccess/GetProductListQuery.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Congo.WebApi.Data.Models;
+using MediatR;
+using System.Collections.Generic;
+
+namespace Congo.WebApi.Data.ProductAccess
+{
+    public record GetProductListQuery() : IRequest<List<Product>>;
```

Comment:
No need for the empty record constructor to be made explicit, so the the parenthesis can be removed.